README.md

```diff
@@ -1,11 +1,13 @@
-# 🚩 Challenge {challengeNum}: {challengeEmoji} {challengeTitle}
+# 🚩 Challenge 4: Minimum Viable Exchange
 
-A {challengeDescription}.
+This challenge will help you build/understand a simple decentralized exchange, with one token-pair (ERC20 BALLOONS ($BAL) and ETH). This repo is an updated version of the original tutorial and challenge repos before it. Please read the intro for a background on what we are building first!
 
 🌟 The final deliverable is an app that {challengeDeliverable}.
 Deploy your contracts to a testnet then build and upload your app to a public web server. Submit the url on [SpeedRunEthereum.com](https://speedrunethereum.com)!
 
-💬 Meet other builders working on this challenge and get help in the {challengeTelegramLink}
+There is also a 🎥 [Youtube video](https://www.youtube.com/watch?v=eP5w6Ger1EQ&t=364s&ab_channel=SimplyExplained) that may help you understand the concepts covered within this challenge too:
```

Comment:
change it to https://www.youtube.com/watch?v=eP5w6Ger1EQ , so it will not point to the end of the video. ab_channel also not needed